.golangci.yml

```diff
@@ -40,6 +36,17 @@ issues:
         - bodyclose
         - funlen
 
+    - path: internal/cmd
+      linters:
+        # using init() is recommended with spf13/cobra
+        - gochecknoinits
+        # using globals is recommended with spf13/cobra
+        - gochecknoglobals
+
+    - path: internal/version
+      linters:
+        - gochecknoglobals
```

Comment:
Why version and cmd are not using nolint directive but a global one?

Reply:
For cmd: because it's part of how cobra works, so I prefer to just disable it.
For version: good point, will use a specific one.

Reply:
https://github.com/nobe4/gh-not/pull/234/commits/eeb51192c3e1a0c07e40b74f1771d66cee23d1d5